ome/omero-figure
# strokeWidth handling

This extends PR https://github.com/ome/omero-figure/pull/409 (@emilroz), so that the Export script handles `shape.get('strokeWidth')` being a string everywhere.
This PR also fixes the origin of the `strokeWidth` string... 

To test Export Script:
 - See https://merge-ci.openmicroscopy.org/web/figure/file/71026/
 - I created this figure with Line, Rectangle, Ellipse, Arrow and Polygon (and scalebar) all with stroke width of 0.75. 
 - Exported the figure to JSON and edited to replace `:0.75,` with `"0.75",` everywhere.
 - Created a new figure with this JSON - If you `File > Export to JSON` you will see this.
 - To test, exported this to PDF and to TIFF.
 - Compare to check they look the same as the original figure.

To test Paste of crop region:
 - Start a new figure (easier)
 - `Copy` the crop region on one panel (zoomed-in) and `Paste` to create a new Rectangle ROI on a different Panel.
 - File > Export to JSON and search for 'strokeWidth' to check that the value not a string e.g. it should be `"strokeWidth":2` NOT `"strokeWidth":"2"`.
